osgar/tools/strip.py

```diff
@@ -0,0 +1,43 @@
+"""
+  Remove some streams from logfile
+"""
+from datetime import timedelta
+
+from osgar.logger import LogReader, LogWriter, lookup_stream_id
+from osgar.lib.serialize import deserialize
```

Comment:
Not needed.

Reply:
I originally deserialized the data, but it is not necessary - fixed